submissions/OlegPopovych/html_css_popup/index.html

```diff
@@ -0,0 +1,142 @@
+			</li>
+			<li class="nav__list--item">
+				<a class="nav__link" href="#">
+					<span>Photos</span>
+				</a>
+			</li>
+			<li class="nav__list--item relative">
+				<input class="popup__check nav__label nav__input" type="checkbox" id="check-1" />
+				<label class="nav__label " for="check-1">
+					<a class="icon" alt="Menu opener">
+						<svg fill="none" height="24" viewBox="0 0 24 24" width="24" class="icon icon-rotate"
+							xmlns="http://www.w3.org/2000/svg">
+							<path fill="#CBD5E4" d="M4 6H20M4 12H20M4 18H20" stroke="#CBD5E4" stroke-linecap="round"
+								stroke-linejoin="round" stroke-width="2" />
+						</svg>
+					</a>
+				</label>
+				<div class="popup">
+					<ul class="popup__menu">
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/blogs.png" alt="Blogs" />
+								Blogs
+							</a>
+						</li>
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/classroom.png" alt="Classroom" />
+								Classroom
+							</a>
+						</li>
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/documents.png" alt="Documents" />
+								Documents
+							</a>
+						</li>
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/files.png" alt="Files" />
+								Files
+							</a>
+						</li>
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/promo.png" alt="Promo" />
+								Promo
+							</a>
+						</li>
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/collection.png" alt="Collection" />
+								Collection
+							</a>
+						</li>
+					</ul>
+					<input class="popup__input" type="checkbox" id="check-2" />
+					<ul class="popup__sub">
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/search.png" alt="Search" />
+								Search
+							</a>
+						</li>
+						<li class="popup__menu--item"><a class="popup__link" href="#">
+								<img class="icon" src="img/chat.png" alt="Chat" />
+								Chat
+							</a>
+						</li>
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/news.png" alt="News" />
+								News
+							</a>
+						</li>
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/maps.png" alt="Mars" />
+								Mars
+							</a>
+						</li>
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/mic.png" alt="Microphone" />
+								Mic
+							</a>
+						</li>
+					</ul>
+					<label class="menu-label popup__sub--toggle" for="check-2">
+						<span class="popup__sub--button">Show more</span>
```

Comment:
Please, hide `Show more` button after clicking

Reply:
Yeah, I hid it, but it is possible to see it again only after reloading the page.

Reply:
It is still not hidden after clicking, I can find it in the bottom of popup